requirements/centos8.requirements.txt

```diff
@@ -1,5 +1,5 @@
 pylint==2.6.0
 astroid==2.4.2
-pytest==6.2.1
+pytest==6.2.5
```

Comment:
pytest was throwing an error when running locally with python `3.10.0`, the fix was upgrading the version to `6.2.5`